chainer/utils/argument.py

```diff
@@ -11,3 +11,7 @@ def parse_kwargs(kwargs, *name_and_values):
         args = ', '.join(["'%s'" % arg for arg in kwargs.keys()])
         raise TypeError('got unexpected keyword argument(s) %s' % args)
     return tuple(values)
+
+
+def assert_kwargs_empty(kwargs):
+    return parse_kwargs(kwargs)
```

Comment:
I think we should not return nothing because we cannot infer what is returned from the function name.